modules/data_pipeline/main.tf

```diff
@@ -11,6 +11,14 @@ data "archive_file" "source" {
     content  = file("{path.module}/../../climateiq-cnn/usl_pipeline/cloud_functions/main.py")
     filename = "main.py"
   }
+  # Add all the contents of cloud_functions/wheels to a /wheels directory inside the zip file.
+  dynamic "source" {
+    for_each = fileset("{path.module}/../../climateiq-cnn/usl_pipeline/cloud_functions/wheels/", "*.whl")
```

Comment:
Are there actually multiple files, or is to avoid having to hard-code a specific file name? Might be worth a comment explaining why there's a for loop despite only expecting one file.